.github/workflows/check-pr-title.yml

```
@@ -0,0 +1,14 @@
+name: Check PR Title
+on:
+  pull_request:
+    branches: [ "main" ]
+    types: [opened, edited, synchronize, ready_for_review, reopened]
+
+jobs:
+  check-pr-title:
+    name: Check PR title
+    runs-on: ubuntu-24.04
+    steps:
+      - uses: blumilksoftware/action-pr-title@e05fc76a1cc45b33644f1de51218be43ac121dd0 # v1.2.0 # https://github.com/blumilksoftware/action-pr-title
+        with:
+          regex: '^(interns2025c\-\d+ |#\d+ )?- .+'
```

```suggestion
          regex: '^(I25C\-\d+ |#\d+ )?- .+'
```
shorter should be better - I think.